tests/Log/LogManagerTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Illuminate\Tests\Log;
+
+use Illuminate\Log\LogManager;
+use Orchestra\Testbench\TestCase;
+
+class LogManagerTest extends TestCase
+{
+    public function testLogManagerCachesLoggerInstances()
+    {
+        $manager = new LogManager($this->app);
+
+        $logger1 = $manager->channel('single')->getLogger();
+        $logger2 = $manager->channel('single')->getLogger();
+
+        $this->assertEquals(spl_object_id($logger1), spl_object_id($logger2));
```

Comment:
This could probably be an $this->assertSame($logger1, $logger2);

Reply:
Just do assertSame on the objects.